modules/vms/versions.tf

```diff
@@ -26,7 +26,7 @@ locals {
     }
     "azure" = {
       "ubuntu-16.04" = "Canonical/UbuntuServer/16.04-LTS/latest"
-      "coreos-1548.3.0" = "CoreOS/CoreOS/Beta/1548.3.0"
+      "azure-latest" = "CoreOS/CoreOS/Beta/latest"
```

Comment:
naming CoreOS latest as azure-latest hides the OS of the base image used.

Reply:
I also don't get this chnge, if this change is not required better to revert